cfy_manager/config.py

```diff
@@ -81,9 +82,34 @@ def dump_config(self):
                     )
                 )
 
+    def _validate_config(self):
```

Comment:
I think it would make more sense to put this code in validations.py

Reply:
Fair point, I'll migrate it.